server/api/posters.py

```diff
@@ -19,10 +20,10 @@ def update_geojson(args: dict, add: bool=True) -> "json":
     are removed.
 
     Parameters:
-    - args (dict): A dictionary containing the following keys:
-        - 'pins' (list of str): A list of pins to add or remove. Each pin is a string in 
+    - args (Dict[str, Any]): A dictionary containing the following keys:
```

Comment:
don't type hint comments, only function headers itself. Typehinting comments doesn't do anything for VSCode's intellisense to suggest autocomplete stuff